src/variable.jl

```diff
@@ -1,4 +1,23 @@
-import .Ops:
+module Variables
```

Comment:
Can you explain to me this increased use of modules?
They seem to make everything harder in julia.
(I wrote a [SO answer about just not using them, ages back](https://stackoverflow.com/a/40314833/179081))
I'm interested in the motivation here.